### PR TITLE
Alpakka Kafka 1.0

### PR DESCRIPTION
**Pull request by 2m**
Please merge only after Alpakka Kafka 1.0 release is done and the docs have been deployed.

### CODE REVIEW

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are happy to announce the general availability of Alpakka Kafka 1.0. The artifacts are the same as the last release candidate 1.0-RC2.
+
+Highlights of the release since 0.22 are:
+
+* Kafka Client has been upgraded to 2.1.0
```
Comment (helena) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: 🥇

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* No more `WakeupException`
+* Alpakka Kafka consumers don’t fail for non-responding Kafka brokers anymore
+* New `Committer.sink` and `Committer.flow` for standardised committing
+* Comitting with metadata
```
Comment (jrudolph) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]:
```suggestion
* Committing with metadata
```

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Java APIs for all settings classes
+* Alpakka Kafka testkit
+
+Some of these changes have been developed or initiated by the community. A big thank you for everyone involved!
```
Comment (helena) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]:
```suggestion
Some of these changes have been developed or initiated by the community. A big thank you to everyone involved!
```

Comment (raboof) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: Would it be possible to generate a list of contributors?

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Highlights of the release since 0.22 are:
+
+* Kafka Client has been upgraded to 2.1.0
+* No more `WakeupException`
+* Alpakka Kafka consumers don’t fail for non-responding Kafka brokers anymore
+* New `Committer.sink` and `Committer.flow` for standardised committing
+* Comitting with metadata
+* Java APIs for all settings classes
+* Alpakka Kafka testkit
```
Comment (raboof) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: Is it possible to link to github issues for some of these? Especially the "No more WakeupException" sounds mysterious if you're not already familiar ;)

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+Some of these changes have been developed or initiated by the community. A big thank you for everyone involved!
+
+For detailed changes, credits, and a bit of the Alpakka Kafka connector history, take a look at the [release notes](https://doc.akka.io/docs/alpakka-kafka/current/release-notes/1.0.html) in the Alpakka Kafka connector documentation.
```
Comment (helena) [inline blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]:
```suggestion
For detailed changes, credits, and more on the Alpakka Kafka connector history, please see the [release notes](https://doc.akka.io/docs/alpakka-kafka/current/release-notes/1.0.html) in the Alpakka Kafka connector documentation.
```

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md (hunk 3 above)
Comment (2m) [reply blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: We do this in the release notes, for example:
https://doc.akka.io/docs/alpakka-kafka/current/release-notes/1.0-RC2.html#general-information
https://doc.akka.io/docs/alpakka-kafka/current/release-notes/1.0-RC1.html#general-information

File: blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md (hunk 4 above)
Comment (2m) [reply blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: We just do a quick overview in the blog post as more detailed information is available in the release notes.

Comment (raboof) [reply blog/_posts/2019-02-28-alpakka-kafka-1.0-released.md]: ah right makes sense didn't notice :+1:

**Pull request merged**